src/data/achievements.js

```diff
@@ -241,6 +242,39 @@ const achievements = [
       inventoryLimit: state.inventoryLimit + reward,
     }),
   }))(),
+
+  ((goal = 500000) => ({
+    id: 'i-am-rich-1',
+    name: 'I am Rich!',
+    description: `Earn ${dollarString(goal)}`,
```

Comment:
Sorry I missed this previously, but these `description` strings will also need a period at the end to be consistent with the other achievements.